src/registrar/templatetags/custom_filters.py

```diff
@@ -48,3 +49,17 @@ def contains_checkbox(html_list):
         if re.search(r'<input[^>]*type="checkbox"', html_string):
             return True
     return False
+
+
+@register.filter
+def get_organization_long_name(organization_type):
+    organization_choices_dict = {}
+
+    for name, value in DomainApplication.OrganizationChoicesVerbose.choices:
+        organization_choices_dict[name] = value
+
+    long_form_type = organization_choices_dict[organization_type]
+    if long_form_type is not None:
+        return long_form_type
+
+    return "Error"
```

Comment:
**suggestion (non-blocking)**: Since this filter is just for organization names, could this be less generic" "Organization type error" maybe?

Reply:
You know. I think in context it does not matter (a user sees Organization: Error vs Organization: Organization type error). But maybe we log a organization type error...